multiple_generate.sh

```diff
@@ -4,17 +4,47 @@ function cloneSubProject() {
   git clone $1 $2 && cd $_
   rm -rf .git
   SUB_ROOT_DIR=$(pwd)
-  cd $3
-  if [ ! -f index.rst ] && [ ! -f index.md ]; then
-    echo -e "No index.rst found at $2/$3. Auto generating...\n"
-    python $BASE/modules/scripts/genindex.py $SUB_ROOT_DIR
-    if [ "${DEBUG:-false}" == "true" ]; then
-      echo -e "\n--------------------------------------\n"
-      cat index.rst
-      echo -e "\n--------------------------------------\n"
-    fi
-    echo -e "Auto generated index.rst\n"
+  # initializing the variables
+  export PROJECTNAME=""
+  export VERSION=""
+  export AUTHOR=""
+  export DEBUG=""
+  export MARKDOWN_FLAVOUR=""
+  export LOGO=""
+  export DOCTHEME=""
+  export DOCPATH=""
+  export AUTOAPI_PYTHON=""
+  export AUTOAPI_PYTHON_PATH=""
+  export AUTOAPI_JAVA=""
+  export AUTOAPI_JAVA_PATH=""
+  export MOCK_MODULES=""
+  export DOCURL=""
+  export HEROKU_APP_NAME=""
+  export SWAGGER_SPEC_URL=""
+  export SWAGGER_UI=""
+  export JAVADOC_PATH=""
+
+  ENVVARS="$(python ${BASE}/modules/scripts/config.py "${USERNAME}" "${REPONAME}")"
```

Comment:
your suggestion for removal of `subproject` sounds good then I'll change to create documentation from sub project from WEBUI . Actually, #297 won't scrape the current implementation. we just register web hook to all the sub project if it triggers we generate documentation based on the config we saved in the DB. merging of this PR won't be a problem.